libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -47,6 +47,19 @@ ebpf_free(_Frees_ptr_opt_ void* memory)
         ExFreePool(memory);
 }
 
+__drv_allocatesMem(Mem) _Must_inspect_result_ _Ret_maybenull_
+    _Post_writable_byte_size_(size) void* ebpf_allocate_cache_aligned(size_t size)
+{
+    return ExAllocatePool2(POOL_FLAG_NON_PAGED | POOL_FLAG_CACHE_ALIGNED, size, EBPF_POOL_TAG);
+}
+
+void
+ebpf_free_cache_aligned(_Frees_ptr_opt_ void* memory)
```

Comment:
why is this needed if its just a wrapper over ebpf_free?

Reply:
For user mode it redirects to _alligned_free